tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Utils/Database.cs

```diff
@@ -18,8 +17,7 @@ public async Task Reset(IConfigurationRoot config)
             await databaseConnection.ExecuteAsync("ALTER ROLE db_datareader ADD MEMBER NHSD;");
             await databaseConnection.ExecuteAsync("ALTER ROLE db_datawriter ADD MEMBER NHSD;");
             await databaseConnection.ExecuteAsync("DELETE FROM Organisations;");
-            await databaseConnection.ExecuteAsync("DELETE FROM AspNetUsers WHERE SecurityStamp = 'TestUser';");
-            await databaseConnection.ExecuteAsync("DELETE FROM AspNetUsers WHERE LockoutEnabled != 1;");
+            await databaseConnection.ExecuteAsync("DELETE FROM AspNetUsers WHERE [Email] NOT IN ('user@agency.com', 'AliceSmith@email.com', 'BobSmith@email.com');");
```

Comment:
Is this case sensitive?

Reply:
This casing is replicating the one in the DB. It feels like an overkill but I suppose I could refactor it to use NormalisedEmail instead.

Thoughts?